unix.c

```diff
@@ -232,13 +269,18 @@ int
 enet_socket_set_option (ENetSocket socket, ENetSocketOption option, int value)
 {
     int result = -1;
+    int _true = 1;
     switch (option)
     {
         case ENET_SOCKOPT_NONBLOCK:
 #ifdef HAS_FCNTL
             result = fcntl (socket, F_SETFL, (value ? O_NONBLOCK : 0) | (fcntl (socket, F_GETFL) & ~O_NONBLOCK));
 #else
+#ifdef HAS_IOCTL
             result = ioctl (socket, FIONBIO, & value);
+#else
+            result = setsockopt (socket, SOL_SOCKET, SO_NONBLOCK, (char * ) & _true, sizeof(int));
```

Comment:
this just hack & copy from old vita code. I don't remember in the old cases, but it wouldn't work properly with `&value`
(maybe just passing the const number makes problem.)

Reply:
I don't understand how `&_true` is different than `&value`. Aren't they both int variables set to 1?

Reply:
hm. an original code of xyzz was also same. I try to do cross checking in my local.